testsuite/tests/shape-index/index_objects.ml

```diff
@@ -0,0 +1,49 @@
+(* TEST
+
+flags = "-bin-annot -bin-annot-occurrences";
+compile_only = "true";
+readonly_files = "index_objects.ml";
+setup-ocamlc.byte-build-env;
+all_modules = "index_objects.ml";
+ocamlc.byte;
+check-ocamlc.byte-output;
+
+program = "-quiet -index -decls index_objects.cmt";
+output = "out_objinfo";
+ocamlobjinfo;
+
+check-program-output;
+*)
+
+let o = object
+  method pop () = ()
+end
+
+(* FIXME: the Typedtree lacks required information to index method usages *)
```

Comment:
It is more than methods are too dynamic, and don't have necessarily a static declaration:
```ocaml
let x = if Random.bool () then object method x = 0 end else object method x = 1 end
```